datastore/google/cloud/datastore/helpers.py

```diff
@@ -471,6 +472,14 @@ def _set_protobuf_value(value_pb, val):
         setattr(value_pb, attr, val)
 
 
+def _call_api(fnc_call, retry, *args, **kwargs):
+
+    call = functools.partial(fnc_call, *args, **kwargs)
+    if retry:
```

Comment:
If `retry` is None, then avoide the partial and just `return fnc_call(*args, **kwargs)`.